assets/apps/customizer-controls/src/form-token-field/FormTokenFieldComponent.js

```diff
@@ -0,0 +1,76 @@
+import { FormTokenField } from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { maybeParseJson } from '@neve-wp/components';
+
+const FormTokenFieldComponent = ({ control }) => {
+	const parsedValue = maybeParseJson(control.setting.get());
+	const [value, setValue] = useState(parsedValue);
+	const { choices, label, description } = control.params;
+
+	const hasLabel =
+		choices !== null &&
+		Array.isArray(choices) &&
+		choices.length > 0 &&
+		choices[0].hasOwnProperty('label');
```

Comment:
This seems unnecessary. The control should always receive the same data format for choices. Especially since these are new controls and we don't have to do any backward compatibility.

In the PRO version, you assign two different array shapes to the choices. The choices should always be the same shape.